packages/mobile/src/tokens/utils.ts

```diff
@@ -19,3 +20,9 @@ export function getHigherBalanceCurrency(
   }
   return maxCurrency
 }
+
+export const sortByUsdBalance = (token1: TokenBalance, token2: TokenBalance) => {
+  const token1UsdBalance = token1.balance.multipliedBy(token1.usdPrice ?? 0)
+  const token2UsdBalance = token2.balance.multipliedBy(token2.usdPrice ?? 0)
+  return token2UsdBalance.minus(token1UsdBalance).toNumber()
+}
```

Comment:
`BigNumber` has a [`comparedTo` method](https://mikemcl.github.io/bignumber.js/#cmp) for this purpose which doesn't require converting to a `number` which could overflow.
also nit: plain old `function` declaration is a few keystrokes shorter than `const` + `arrow function` 😄 
```suggestion
// Sort by usd balance (descending)
export function sortByUsdBalance(token1: TokenBalance, token2: TokenBalance) {
  const token1UsdBalance = token1.balance.multipliedBy(token1.usdPrice ?? 0)
  const token2UsdBalance = token2.balance.multipliedBy(token2.usdPrice ?? 0)
  return token2UsdBalance.comparedTo(token1UsdBalance)
}
```